test/module/irohad/validation/chain_validation_test.cpp

```diff
@@ -135,24 +119,19 @@ TEST_F(ChainValidationTest, FailWhenNoSupermajority) {
  */
 TEST_F(ChainValidationTest, ValidWhenValidateChainFromOnePeer) {
   // Valid previous hash, has supermajority, correct peers subset => valid
-  auto block = std::make_shared<shared_model::proto::Block>(getBlockBuilder().build());
-  rxcpp::observable<std::shared_ptr<shared_model::interface::Block>>
-      block_observable = rxcpp::observable<>::just(block).map([](auto &&x) {
-        return std::shared_ptr<shared_model::interface::Block>(x);
-      });
+  auto block = std::make_shared<BlockMock>();
```

Comment:
Upwards value, here pointer. Is this done for purpose?

Reply:
Sure, there's need in pointer at 135 line